frontend/src/app/application-forms/fields/file-upload.component.ts

```diff
@@ -108,9 +99,29 @@ export class FileUploadComponent implements DoCheck, OnInit {
     }
   }
 
-  clickInput(event) {
+  clickInput(event, status, index) {
```

Comment:
Nice work consolidating the click handlers into one method. Given the contents of the switch statement it seems pretty implicit what the method is for, but perhaps we could come up with a more meaningful/relevant name than `clickInput`?

Reply:
@briandavidson - How does `FileUploadDeleteHandler()` sound?

Reply:
That's more descriptive, for sure. I'll say g2g since this is pretty minor, but if you wanna go the extra mile it might be worth seeing if @mtlaney has any better suggestions.